tests/models/test_library.py

```diff
@@ -1,81 +1,100 @@
 # encoding: utf-8
 import pytest
-from ...testing import DatabaseTest
 from ...model.configuration import ConfigurationSetting
 from ...model.hasfulltablecache import HasFullTableCache
 from ...model.library import Library
 
-class TestLibrary(DatabaseTest):
 
-    def test_library_registry_short_name(self):
-        library = self._default_library
+class TestLibrary:
+
+    def test_library_registry_short_name(self, db_session, create_library):
+        """
+        GIVEN: A Library
+        WHEN:  The short name is set to "foo"
+        THEN:  The short name should be set to "FOO"
+        """
+        library = create_library(db_session)
 
         # Short name is always uppercased.
         library.library_registry_short_name = "foo"
         assert "FOO" == library.library_registry_short_name
 
         # Short name cannot contain a pipe character.
-        def set_to_pipe():
+        with pytest.raises(ValueError):
             library.library_registry_short_name = "foo|bar"
-        pytest.raises(ValueError, set_to_pipe)
 
         # You can set the short name to None. This isn't
         # recommended, but it's not an error.
         library.library_registry_short_name = None
+        assert None == library.library_registry_short_name # Do we need this assert?
```

Comment:
Yeah, I think so -- it lets us verify that this isn't silently implemented as a no-op or something.